docs/api/single_graph.rst

```diff
@@ -1,7 +1,7 @@
 SingleGraph
 ============
 
-``SingleGraph``s convert a molecule into a SMIRKS pattern by
+``SingleGraph``\'s convert a molecule into a SMIRKS pattern by
```

Comment:
I don't think the apostrophe here is grammatically correct, but you're right to highlight the poor rendering

https://chemper.readthedocs.io/en/latest/docs/api/single_graph.html

![image](https://github.com/user-attachments/assets/e20c5f68-0129-4888-9aa5-f23827633e6b)

Reply:
thanks agreed 🙈